apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -239,6 +252,42 @@ export default connect(
           return style.debugOutputBackgroundWarning;
       }
     }
+    projection() {
+      return geoMercator()
+        .scale(60)
+        .translate([400 / 2, 400 / 2]);
+    }
+
+    async mercator() {
+      let displayWorldMap = async function() {
+        const worldMapUrl = 'https://unpkg.com/world-atlas@1/world/110m.json';
+        let response = await fetch(worldMapUrl);
+
+        let worldData = await response.json();
+
+        return feature(worldData, worldData.objects.countries).features;
+      };
+
+      let worldMap = await displayWorldMap();
+      console.log('world2', worldMap);
+
+      return (
+        <svg width={400} height={400}>
+          <g className="countries">
+            {worldMap.map((d, i) => (
+              <path
+                key={`path-${i}`}
+                d={geoPath().projection(this.projection())(d)}
+                className="country"
+                fill={'green'}
```

Comment:
At some point, we'll want to consult with product/design to choose a color scheme for this feature, but that's not something that needs a decision now.

Reply:
Ok, I'll leave a TODO comment in there to remind myself!